tests/test_integration.py

```diff
@@ -18,22 +18,31 @@
 
 
 def make_factory():
-    automap = automap_base()
+    # NOTE: Due to the changes made in the production database and the switch
+    # to using views instead of tables, we have attempted to reconstruct the
+    # schema with our test data.  It is an approximation of what is actually
+    # in production, but we are focused more on ensuring that the behavior and
+    # processing of our scripts and business logic are sound as opposed to
+    # mirroring the data exactly as it is at this time.
+    metadata = sa.MetaData(schema='disclosure')
+    automap = automap_base(bind=db.engine, metadata=metadata)
     automap.prepare(db.engine, reflect=True)
 
     class SchedAFactory(SQLAlchemyModelFactory):
         class Meta:
             sqlalchemy_session = db.session
-            model = automap.classes.fec_vsum_sched_a
-        filing_form = '11'
+            model = automap.classes.nml_sched_a
```

Comment:
Yeah, this was a bit tricky but simply required setting the `automapper` to a different schema.  What's a bit of a bummer about that you only have access to the one schema.  I suppose you could alleviate that with multiple `automapper` objects though, now that I think about it.

I'm with you on being wary about not having a completely accurate reflection of the databases as they actually exist, but I noticed that for these views, `disclosure.v_sum_and_det_sum_report` was only being referenced for 3 columns; the rest of the data came from `disclosure.nml_sched_x`.

However, those 3 columns represent `filing_form`, `rpt_tp`, and `rpt_yr`.  Would it be this table that actually receives more updates once a record exists in the corresponding `disclosure.nml_sched_x` table?  If so, then we either need to change the triggers again or have them run off of `disclosure.v_sum_and_det_sum_report` as well to ensure we always receive record updates when they occur.  This leads me to my own question:  is this how record amendments are managed?  If so, ugh, back to the drawing board a bit more...

As for when the rows are actually inserted into the `ofec_sched_x` tables (whichever child table they should actually go into), that is handled by the nightly update process:

1. First, [`manage.update_aggregates`](https://github.com/18F/openFEC/blob/develop/manage.py#L255) is what gets kicked off
1. [`partition.base.refresh_children`](https://github.com/18F/openFEC/blob/develop/webservices/partition/base.py#L210) is called for Schedule A and B as a part of that, which loops through all child tables
1. [`partition.base.refresh_child`](https://github.com/18F/openFEC/blob/develop/webservices/partition/base.py#L219) actually does the legwork for each child table, and clears out the processed records from the queues when done

At the end, both queues should be completely empty and our child tables updated with fresh data. :-)